static/js/message_store.js

```diff
@@ -118,8 +118,8 @@ function add_message_metadata(message) {
         composebox_typeahead.add_topic(message.stream, message.subject);
         message.reply_to = message.sender_email;
 
-        stream_data.process_message_for_recent_topics(message);
         set_topic_edit_properties(message);
+        stream_data.process_message_for_recent_topics(message);
```

Comment:
Yep, seems like it was me doing random changes then `git add`-ing them, just in case... sorry for this one. :smiley: